sormas-api/src/main/resources/captions_ar.properties

```diff
@@ -1134,6 +1134,7 @@ statisticsDataDisplayed = Data displayed
 symptomsLesionsLocations = Localization of the lesions
 symptomsMaxTemperature = Maximum body temperature in ° C
 symptomsSetClearedToNo = Set cleared to No
+symptomsSetClearedToUnknown=Set to Unknown
```

Comment:
Revert changes to all captions_XX files that aren't the original captions.properties file, please. Those are handled via the translation tool.